src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,102 @@
+package duke;
+
+public class Ui {
+
+    /**
+     * Constructor to create a UI object.
+     */
+    Ui() {}
+
+    private final static String LOGO = " ____        _        \n"
+                                    + "|  _ \\ _   _| | _____ \n"
+                                    + "| | | | | | | |/ / _ \\\n"
+                                    + "| |_| | |_| |   <  __/\n"
+                                    + "|____/ \\__,_|_|\\_\\___|\n";
+
+    private final static String HORIZONTAL_DIVIDE =
+        "___________________________________________________________________";
+
+    /**
+     * Prints out a welcome message.
+     */
+    public void welcomeUser() {
+        System.out.println("Hello from\n" + LOGO);
+        System.out.println("What can Duke do for you today?");
+        System.out.println(HORIZONTAL_DIVIDE);
+    }
+
+    /**
+     * Prints out the added entry.
+     *
+     * @param entry THe newly-added entry.
+     * @param id The #id of the newly-added entry.
+     */
+    public void addEntry (Entry entry, int id) {
+        System.out.println("I've added entry to your list!");
+        this.printEntry(entry, id);
+        System.out.println(HORIZONTAL_DIVIDE);
+    }
+
+    /**
+     * Prints out a horizontal line at the end of some operation.
+     */
+    public void endCommand() {
+        System.out.println(HORIZONTAL_DIVIDE);
+    }
+
+    public void listMatches() {
+        System.out.println("Looking for matching tasks in your list..");
+    }
+
+    public void foundMatches() {
+        System.out.println("Here are the matching tasks in your list..");
+    }
+
+    /**
+     * Prints out given entry.
+     *
+     * @param entry Entry to be printed.
+     * @param id ID of entry to be printed.
+     */
+    public void printEntry(Entry entry, int id) {
+        System.out.println("\t" + id + "." + entry);
+    }
+
+    /**
+     * Prints out farewell message.
+     */
+    public void goodByeUser() {
```

Comment:
This method's name could potentially be improved as method names should be verbs. Perhaps something like 'printGoodByeMessage'.

Reply:
resolved, thanks!